bin/pygrb/pycbc_pygrb_efficiency

```diff
@@ -606,7 +612,17 @@ if do_injections:
             excl_dist = 0
             msg = "Efficiency below %d%% in first bin!" % (percentile)
             logging.warning(msg)
-        # TODO: include percentile, excl_dist on output pages
+
+    # Write 50% and 90% exclusion distances to JSON file
+    # Also include injection set name and trial name
+    if opts.exclusion_dist_output_file:
+        excl_dist_dict = {}
+        excl_dist_dict['inj_set'] = inj_set_name
```

Comment:
The user must provide `opts.injection_set_name` if `opts.exclusion_dist_output_file` and/or `opts.found_missed_file` are provided.
Looking at this made me wonder about something I had not caught in the past: does it even make sense to have `opts.found_missed_file` and `opts.found_missed_file` as optional?  I.e., if `do_injections` and/or `onsource_file` are False, does the code do something meaningful (and not crash)?  Otherwise we can promote those options to required.

Reply:
The injections part of the code uses results from the onsource analysis, so I'd say it wouldn't hurt anything to make both files required, but there could be cases I'm not considering.